app/src/main/java/io/intrepid/thirdpartylibraryexcercise/MainActivity.java

```diff
@@ -1,17 +1,29 @@
 package io.intrepid.thirdpartylibraryexcercise;
 
 import android.app.ProgressDialog;
+import android.content.Context;
 import android.graphics.Bitmap;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
 import android.view.View;
 import android.widget.Button;
 import android.widget.ImageView;
+import android.widget.Toast;
```

Comment:
My fault again, but there are a bunch of unused imports here. Just `alt+enter` with the cursor over it and it should give you an option to remove them